Assets/Scripts/GameManager.cs

```diff
@@ -44,12 +44,14 @@ public class GameManager : MonoBehaviour {
         _canvas = GameObject.Find("Canvas");
         AddColorsAndAnchorsToList();
         //Create the players
-        CreateHumanPlayer("Player1 Tank", new Vector3(-5, 0, 0), "HorizontalKeyboard", "VerticalKeyboard", "FireKeyboard",
-            "RotateTurretController1", false);
-        CreateHumanPlayer("Player2 Tank", new Vector3(5, 0, 0), "HorizontalController1", "VerticalController1", "FireController1",
-            "RotateTurretController1", true);
-        CreateHumanPlayer("Player3 Tank", new Vector3(0, 3, 0), "HorizontalController2", "VerticalController2", "FireController2",
-            "RotateTurretController2", true);
+        CreateHumanPlayer("Player1 Tank", new Vector3(-5, 0, 0), "HorizontalKeyboard1", "VerticalKeyboard1",
```

Comment:
`HorizontalKeyboard1`, `VerticalKeyboard1`, etc as final static strings (or whatever the C# equivalent is)?